libpod/container_internal_linux.go

```diff
@@ -752,8 +752,24 @@ func (c *Container) makeBindMounts() error {
 
 // generateResolvConf generates a containers resolv.conf
 func (c *Container) generateResolvConf() (string, error) {
+	resolvConf := "/etc/resolv.conf"
+	for _, ns := range c.config.Spec.Linux.Namespaces {
+		if ns.Type == spec.NetworkNamespace {
+			if ns.Path != "" {
```

Comment:
Also check that path does not start with /proc/pid/ns - I don't think we want to do this in that case

Reply:
Fixed,  I just check for HasPrefix("/proc")